src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/RegionHighCourtFrc.java

```diff
@@ -1,16 +1,16 @@
 package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd;
 
 import com.fasterxml.jackson.annotation.JsonValue;
-import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
 import com.fasterxml.jackson.databind.annotation.JsonNaming;
 import lombok.RequiredArgsConstructor;
 
 import java.util.Arrays;
 
-@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
 @RequiredArgsConstructor
 public enum RegionHighCourtFrc {
-    HIGHCOURT("highcourt");
+    HIGHCOURT("FR_highCourtList_1");
```

Comment:
Whats reason to change this?

Reply:
Its production bug

`Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is java.lang.IllegalArgumentException: Cannot deserialize value of type `uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.RegionHighCourtFrc` from String "FR_highCourtList_1": not one of the values accepted for Enum class: [highcourt]
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseData["directionDetailsCollection"]->java.util.ArrayList[0]->uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.DirectionDetailCollection["value"]->uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.DirectionDetail["localCourt"]->uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.Court["highCourtList"])] with root cause
2023-10-18T18:06:09.094 WARN  [*** ccdDataStoreApi] u.g.h.ccd.domain.service.callbacks.CallbackService Unable to connect to callback service http://localhost:9000/case-orchestration/ccdMidEvent because of InternalServerError 500 : "{"timestamp":"2023-10-18T17:06:09.091+00:00","status":500,"error":"Internal Server Error","trace":"java.lang.IllegalArgumentException: Cannot deserialize value of type `uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.RegionHighCourtFrc` from String \"FR_highCourtList_1\": not one of the values accepted for Enum class: [highcourt]\n at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseData[\"directionDetailsCollection\"]->java.util.ArrayList[0]->uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.DirectionDetailCollection[\"value\"]->uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.DirectionDetail[\"localCourt\"]->uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.Court[\"highCourtList\"])\n\tat hostApplication//com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:4393)\n\tat hostApplication//com.fasterxml.jackson.databind.ObjectMapper.convertValue(ObjectMapper.java:4324)\n\tat uk.gov.hmcts.reform.finrem.caseorchestration.mapper.FinremCaseDetailsMapper.mapToFinremCaseDetails(FinremCaseDetailsMapper.java:23)\n\tat uk.gov.hmcts.reform.finrem.caseorchestration.handler.FinremCallbackHandler.mapToFinremCallbackRequest(FinremCallbackHandler.java:36)\n\tat uk.gov.hmcts.reform.finrem.caseorchestration.handler.FinremCallbackHandler.handle(FinremCallbackHandler.java:24)\n\tat uk.gov.hmcts.reform.finrem.caseorchestration.service.CallbackDispatchService.dispatchToHandlers(CallbackDispatchService.java:45)\n\tat `